src/components/slider/slider.html

```diff
@@ -0,0 +1,14 @@
+<div class="bx--form-item" style="width: 600px;">
```

Comment:
I found that without the inline style, the whole Slider collapses on itself.  And we shouldn't add a `width` to the `.bx--form-item` scss file...need a way to scope it to Slide only.